json/schema/singleSet.schema.json

```diff
@@ -0,0 +1,305 @@
+{
+  "additionalProperties": false,
+  "description": "Validation schema for MTGJSON for single sets (ex: HOU.json && HOU-x.json)",
+  "properties": {
+    "alternativeNames": {
+      "items": {
+        "type": "string"
+      },
+      "type": "array"
+    },
+    "block": {
+      "type": "string"
+    },
+    "booster": {
+      "items": {
+        "anyOf": [
+          {
+            "type": "string"
+          },
+          {
+            "items": {
+              "type": "string"
+            },
+            "type": "array"
+          }
+        ]
+      },
+      "type": "array"
+    },
+    "border": {
+      "type": "string"
+    },
+    "cards": {
+      "items": {
+        "additionalProperties": false,
+        "properties": {
+          "artist": {
+            "type": "string"
+          },
+          "border": {
+            "type": "string"
+          },
+          "cmc": {
+            "type": "number"
+          },
+          "colorIdentity": {
+            "items": {
+              "type": "string"
+            },
+            "type": "array"
+          },
+          "colors": {
+            "items": {
+              "type": "string"
+            },
+            "type": "array"
+          },
+          "flavor": {
+            "type": "string"
+          },
+          "foreignNames": {
+            "items": {
+              "additionalProperties": false,
+              "properties": {
+                "language": {
+                  "type": "string"
+                },
+                "multiverseid": {
+                  "type": "integer"
+                },
+                "name": {
+                  "type": "string"
+                }
+              },
+              "type": "object"
+            },
+            "type": "array"
+          },
+          "hand": {
+            "type": "integer"
+          },
+          "id": {
+            "type": "string"
+          },
+          "imageName": {
+            "type": "string"
+          },
+          "layout": {
+            "type": "string"
+          },
+          "legalities": {
+            "items": {
+              "additionalProperties": false,
+              "properties": {
+                "format": {
+                  "type": "string"
+                },
+                "legality": {
+                  "type": "string"
+                }
+              },
+              "type": "object"
+            },
+            "type": "array"
+          },
+          "life": {
+            "type": "integer"
+          },
+          "loyalty": {
+            "type": [
+              "integer",
+              "null"
```

Comment:
Put it as null due to Nissa with the "X", we will want to convert this field to a string thought